catalog/spatial/csw/spatial-csw-transformer/pom.xml

```diff
@@ -32,6 +32,7 @@
         <dependency>
             <groupId>com.thoughtworks.xstream</groupId>
             <artifactId>xstream</artifactId>
+            <version>1.4.9</version>
```

Comment:
Do we have a constant or dependency management for this somewhere. I don't think we want it to get out of sync.